README.md

```diff
@@ -4,37 +4,64 @@
 in possession of the ball and rotates positions clockwise.
 
+##Officiating
+Jollyballs is first and foremost a self-officiated sport.  It is an expectation that players recognize when they have crossed the line, hit the ball twice, or broken the rule.  In the event that either side could have been at fault and it is uncertain whom is at fault, a re-do should be declared.  
+
+In especially competitive matches, it is not uncommon to have a line judge present.  The line judge has the final say on line infractions and any other contentious plays.  It is important for the line judge to sit along the net so that they have the optimal vantage point to officiate the game.
+
+## End of Game/Match:
+To end the game, the serving team must first state the score then announce "Jolly Point."  Without the declaration of "Jolly Point," the receiving team can declare the point invalid and request a re-do.  On match point, if a team does not declare "Super Jolly Point" or "Jolliest Point" then the same rules apply for "Jolly Point."
+
+At the conclusion of the game, players of both sides line up and shake hands or exchange a words. This is an expected and time-honored tradition.
+
+*KUBA*
+An alternative way to end the game is to achieve a "KUBA."  A "KUBA" is a particularly difficult location, obstacle, or receptacle to have the ball travel through or fall into to immediately end the game.  A prime example is a distant bowl, box, or office basketball hoop. On every court an especially difficult location should be specified for each side to act as the "KUBA."  
+
+## Round Robin Rules:
+* If any multiple of 3 greater than 6 players all would like to play at the same time, a round robin of Jollyballs can be initiated.  In this scenario, the first three players to motion for Jollyballs become the captains following a snake draft selection order.  
+
+* In lieu of traditional matches of three games, teams will rotate sides of the table and opponents after each individual game, rotating from one side of the table, to the other side, then to off the table.  If there is a tie between two teams for being eliminated off the table, the tie breaker will be decided by the total number of points between their two games.
+
 ## Additional Rules:
-* In Jollyballs, all surfaces, with the exception of the floor are live. If the ball hits off the wall, ceiling, tabletop, or any other nearby surface; the ball is playable.
+* In Jollyballs, all surfaces, with the exception of the floor are live. If the ball hits off the wall, ceiling, tabletop, or any other nearby surface; the ball is playable.  If the ball comes to a rest in an accessible place, you have less than three seconds to strike the ball.
 
-* A player may hit the ball off of their body but cannot carry, toss, or throw the ball.
+* A player may hit the ball off of their body but cannot carry, toss, or throw the ball.  Hitting the ball off of your body is known as a Scoop and does not count as a hit.
```

Comment:
Oh, I question whether the scoop should be counted as a hit.  I can get behind it if it's a block, but could be dangerous not counting it normally as a hit.